src/main/kotlin/minesweeper/domain/CellManager.kt

```diff
@@ -17,5 +21,25 @@ class CellManager(
         if (_cells.removeIf { it.position == cell.position }) _cells.add(cell)
     }
 
-    fun getSize() = cells.size
+    fun openAll() {
+        cells.forEach(::openCell)
+    }
+
+    private fun openCell(cell: Cell) {
+        if (cell.isMine().not()) {
+            val aroundMineCount = cell.getAroundPositions().sumBy(::getMineCounter)
+            changeCell(cell.open(aroundMineCount))
+        }
+    }
+
+    private fun getMineCounter(position: Position) =
+        if (getCell(position)?.isMine() == true) 1 else 0
+
+    private fun getCell(position: Position) =
+        cells.find { it.isPosition(position) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        _cells.find { it.isPosition(position) }
```

Reply:
class 내부에서도, 내부의 값을 수정하는 목적이 아니라면, 
_cells보단 cells를 호출하는게 좀더 명확하지 않을까라는 생각이 들었는데, 
어떻게 생각하시나요? 
클래스 내부에선 수정의 유무를 떠나서 _cells를 사용해야할지 궁금합니다